.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Are you using the [Live Server VS Code extension](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)? Generally it's good not to include .vscode files in the repository. You can add this hidden folder to `.gitignore`

Reply:
First of all, I'm sorry. I don't understand how can I do it? Can you please explain me please?I already tried adding the folder to the .gitignore file using copy paste. But it didn't work.